src/lib/commands/SendCommand.ts

```diff
@@ -78,9 +78,22 @@ export class SendCommand extends ExistingNewsLetterCommand {
     return ready;
   }
 
+  private extractEmail() {
+    let email = this.source;
+    if (typeof(email) === "string") {
```

Comment:
can email be not a string? it's typed a string in TS. If it's not a string sometimes, it's a bug.

Reply:
I see what you mean. Fixed.